modules/eigen/5.0.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://gitlab.com/libeigen/eigen/-/archive/5.0.0/eigen-5.0.0.tar.gz",
```

Comment:
See #4364.  GitLab URLs are generally not checksum-stable over time, so should be avoided.  (Even worse than GitHub archive URLs.)  I suggest using https://github.com/eigen-mirror/eigen/archive/refs/tags/5.0.0.tar.gz (from https://github.com/eigen-mirror/eigen/releases/tag/5.0.0), instead.

Reply:
updated!